src/client/firefox/commands.js

```diff
@@ -272,11 +272,13 @@ async function getFrameScopes(frame: Frame): Promise<*> {
 
 function pauseOnExceptions(
   shouldPauseOnExceptions: boolean,
-  shouldIgnoreCaughtExceptions: boolean
+  shouldPauseOnCaughtExceptions: boolean
 ): Promise<*> {
   return threadClient.pauseOnExceptions(
     shouldPauseOnExceptions,
-    shouldIgnoreCaughtExceptions
+    // Providing opposite value because server
+    // uses "shouldIgnoreCaughtExceptions"
+    !shouldPauseOnCaughtExceptions
```

Comment:
The way I'm thinking about this, we can change everything on our side, but simply reverse the value we're using when sending to server since the server still works on ignoreCaughtExceptions.